scripts/check-clippy-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Would be nice if we ate our own dog food here.  I know there can be chicken/egg issues but we are building a shell so we should be able to write simple scripts with it.  Or fix it if we can not...

Reply:
Agreed!